iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService(iox::capro::ServiceDescription(), instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    senderRuntime->stopOfferService(iox::capro::ServiceDescription());
+}
+
+TEST_F(RoudiFindService_test, ReOfferSingleMethodServiceSingleInstance)
+{
+    senderRuntime->offerService({"service1", "instance1"});
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService({"service1", "instance1"}, instanceContainer);
```

Comment:
The lines with `findService` and verifying the contents of the `instanceContainer` could be skipped since you already covered that in a previous tests. Try to follow the pattern
1. arrange
2. act
3. assert

https://wiki.c2.com/?ArrangeActAssert

Reply:
i think removing of findService here will be against the scope of the test, where the scope is to check sequence(start->stop-startOfferService), yeah i should change the name but

Reply:
The lines with findService and verifying the contents of the instanceContainer are removed